firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -140,7 +141,7 @@ uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
 
   // Checks if the EM is aligned properly
   if((currentX % stepsPerUnitSpace) || (currentY % stepsPerUnitSpace)) {
```

Comment:
if statements should be written as:
if (condition)
{

}

You need to put the one bracket on a newline and add a space after "if"